src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_BYE = "bye";
+
```

Comment:
Can diversify the commands adapting the different forms of the same commands(ie cases with capitalized letters